### PR TITLE
Consider `pt_BR`

### PR DESCRIPTION
**Pull request by boscohyun**
SSIA

### CODE REVIEW

File: src/renderer/i18n/index.tsx [hunk 1]
```diff
@@ -40,7 +40,13 @@ export function LocaleProvider({ children }: React.PropsWithChildren<{}>) {
...
-    locale.startsWith(lang.code)
-  );
+  return languages.some((lang: Record<"code", string>) => {
+    let code = lang.code;
+    if (code === "pt_BR")
+    {
+      code = "pt-BR";
+    }
+
+    return locale.startsWith(code);
```
Comment (riemannulus) [inline src/renderer/i18n/index.tsx:50]: IMO this problem will happen with other `Unix style tagging`. How about change this code like...
```suggestion
    return locale.startsWith(lang.code.replace('_', '-'));
```

**Review by riemannulus — COMMENTED** [review]
LGTM but

File: src/renderer/i18n/index.tsx (hunk 1 above)
Comment (boscohyun) [reply src/renderer/i18n/index.tsx]: That's batter.😁

**Review by riemannulus — APPROVED** [review]
👍

**Review by moreal — APPROVED** [review]

**Pull request merged**